### PR TITLE
Cloudsql auth proxy

### PR DESCRIPTION
**Pull request by seslattery**

### CODE REVIEW

File: pachyderm/values.yaml [hunk 1]
```diff
@@ -145,6 +145,22 @@ pachd:
...
+      enabled: true
+      image:
+        repository: "gcr.io/cloudsql-docker/gce-proxy"
+        tag: "1.23.0"
+      resources:
+        requests:
+          # The proxy's memory use scales linearly with the number of active
+          # connections. Fewer open connections will use less memory. Adjust
+          # this value based on your application's requirements.
+          memory: "2Gi"
```
Comment (chainlink) [inline pachyderm/values.yaml:158]: We generally haven't hardcoded requests and limits and left that as an exercise for the user depending on their cluster environment.

**Review by chainlink — CHANGES_REQUESTED** [review]
One small comment, but otherwise looking good! Can you add a section to the GCP test to exercise these values?

**Review by chainlink — APPROVED** [review]
Looks good!

**Comment by seslattery** [comment]
> One small comment, but otherwise looking good! Can you add a section to the GCP test to exercise these values?

Will add test soon, going to e2e this on Hub before I write the test.

**Comment by seslattery** [comment]
I've moved the proxy over to a deployment/service, and wired up workload identity. (I still owe a test too)  @chainlink PTAL

**Comment by chainlink** [comment]
Going to close this one out, as it's merged into the main repo

**Pull request closed without merging**